tests/test_cases/test_cocotb/test_doctests.py

```diff
@@ -2,12 +2,14 @@
 # Licensed under the Revised BSD License, see LICENSE for details.
 # SPDX-License-Identifier: BSD-3-Clause
 import doctest
+import os
 
 import cocotb
 
 
 @cocotb.test()
 async def test_utils(dut):
+    os.environ['COCOTB_ANSI_OUTPUT'] = "0"
```

Comment:
A brief comment about why this is needed would be nice

Reply:
Done.